plugnpy/check.py

```diff
@@ -16,10 +16,9 @@ class Check(object):
     """
     STATUS = {0: 'OK', 1: 'WARNING', 2: 'CRITICAL', 3: 'UNKNOWN'}
 
-    def __init__(self, state_type="METRIC", sep=', ', si_bytes_conversion=False):
+    def __init__(self, state_type="METRIC", sep=', '):
```

Comment:
rename `sep` to `separator`?

Reply:
Also, do we have documentation RE the custom separator

Reply:
@ojenkins I chose sep to replicate Python's built in print() functions 'sep' keyword